mocks/mock-bgs-api/src/main/resources/init-after-tomcat-starts.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+until [ "`curl --silent --show-error --connect-timeout 1 -I http://localhost:8080/castlemock/ | grep 'HTTP/1.1 200'`" != "" ];
```

Comment:
To get the status code, try this more reliable and direct approach instead -- see https://superuser.com/questions/272265/getting-curl-to-output-http-status-code/442395#442395 and https://github.com/department-of-veterans-affairs/abd-vro/blob/89aa4b9ad4eae5eff02e2b1b075345f155edcd5d/.github/workflows/xample-integration-test.yml#L49-L52:
```suggestion
until [ "$(curl --silent -o /dev/null -w "%{http_code}" --connect-timeout 1 -I http://localhost:8080/castlemock/)" == "200"
```